workshops/image_editor/README.md

```diff
@@ -0,0 +1,451 @@
+---
+name: 'Image Editor'
+description: 'Make your own image editor'
+author: '@wollygfx'
+img: 'https://cloud-81e0x4zt3.vercel.app/0image.png'
+---
+
+In this workshop you'll learn how to make a very simple image editor using [CSS Filters](https://www.w3.org/TR/filter-effects/) , [jQuery](https://jquery.com) and [Bootstrap](https://getbootstrap.com/docs/4.5/getting-started/introduction/) . Follow along with me and see how easy it is!
+
+Here's a [live demo](https://image-editor.wollygfx.repl.co) and the [source code](https://repl.it/@wollygfx/Image-editor) if you want to check it out before we start.
+![live demo](https://cloud-gvno5b2im.vercel.app/0screen_recording_2020-11-09_at_8.14.57_pm.gif)
+
+## Set Up
+This workshop requires a basic knowledge of the following languages: HTML, CSS & JS. Don’t worry if you get stuck at some point in the workshop, everything is explained the best way for you to understand!
+
+For this workshop we will use [Repl.it](https://repl.it/), a free, online code editor. Click [here](https://repl.it/languages/html) to get started with a new HTML project on repl.it.
+
+![Repl.it](https://cloud-qbmylslty.vercel.app/0image.png)
+
+## HTML
+To begin, we're going to make all the elements that will be in our image editor. The layout will look like this:
+
+![Image editor's layour](https://cloud-lrjvj5rqz.vercel.app/0image.png)
+1. In **section 1** we will put an image input and a button that updates the image displayed in section 2.
+2. In **section 2** the chosen image will be displayed.
+3. In **section 3** we will put some inputs that will let the user edit the image displayed in section 2.
+
+### Coding the layout
+We're going to use [Boostrap](https://getbootstrap.com), a library that makes CSS styling easy, to easily make this layout.
+
+Add the following code inside of your `<head>` tag. This links the HTML file to the Bootstrap CSS file, and it'll give our app a little bit of cool styling without us having to do it ourselves.
+
+```html
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootswatch/4.5.2/cerulean/bootstrap.min.css" integrity="sha384-3fdgwJw17Bi87e1QQ4fsLn4rUFqWw//KU0g8TvV6quvahISRewev6/EocKNuJmEw" crossorigin="anonymous">
+```
+[Bootstrap’s grid system](https://getbootstrap.com/docs/4.1/layout/grid/) uses a series of containers, rows, and columns to layout and align content. Our website will be full of them:
+![Div Containers](https://cloud-e8d17xj1f.vercel.app/0image.png)
+- Color grey represents the ```<body>``` tag
+- Color green represents the main container.
+- Color pink represents a row.
+- Color blue represents a column, the two last rows are divided by 3 columns.
+
+Add the following code inside your `<body>` tag:
```

Comment:
Here you're transitioning from explaining background info to actually coding, so you need to add some sort of transition. Maybe move `### Coding the layout` here and remove the one above